src/Sfa.Tl.Matching.Application/Mappers/OpportunityMapper.cs

```diff
@@ -26,13 +26,20 @@ public OpportunityMapper()
 
             CreateMap<EmployerNameDto, Opportunity>()
                 .ForMember(m => m.EmployerName, o => o.MapFrom(s => s.CompanyName))
+                .ForMember(m => m.EmployerContact, o => o.MapFrom(s => s.EmployerContact))
```

Comment:
it is not the responsibility of Find Employer Page to Store Employer Contact Name, that's why I removed it,
Instead add another field in database for EmployerCrmId into Opportunity Table and do a get to Employer using this Guid
I know this will mean we will have to get employer details twice but its much cleaner code